packages/cli/src/commands/org/config/preview.ts

```diff
@@ -37,6 +37,12 @@ export default class Preview extends Command {
       default: cwd(),
       required: false,
     }),
+    showResourcesToDelete: flags.boolean({
+      char: 'd',
+      description: 'Whether or not to show resources to delete',
+      default: false,
```

Comment:
Instead of setting a default here, could we instead rely on "the default of the API" ?

So, if the flag is not specified, we just don't pass anything to the API, and it will decide if a deletion should happen or not.